tests/TestCase.php

```diff
@@ -35,7 +35,7 @@ protected function tearDown(): void
     /**
      * @param \Illuminate\Foundation\Application $app
      */
-    protected function getEnvironmentSetUp($app)
+    protected function getEnvironmentSetUp($app): void
```

Comment:
why there is no type for parameter?

Reply:
Must be compatible with Orchestra\Testbench\TestCase getEnviromentSetUp and getPackageProviders